content/participants/ttjc@sushilburagute/index.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: Sushil Buragute
-type: "participant"
```

Comment:
I know you have used some ctrl+C and ctrl+V but you should have changed this to "participants" 😂😂😂